lib/better_errors.rb

```diff
@@ -143,4 +148,6 @@ def self.default_editor
   BetterErrors.binding_of_caller_available = false
 end
 
+BetterErrors.maximum_variable_inspect_size ||= 100_000
```

Comment:
I’m wondering if setting the default value should be moved to line 53, right after setting the default for `ignored_instance_variables`?

I realize that the value of `binding_of_caller_available` is being set down here, but that is probably just to avoid having `require`s inside the module code.